util/settings/accounts.go

```diff
@@ -0,0 +1,307 @@
+package settings
```

Comment:
General thought on this package - I think all Public methods and types should be rather well commented.

Reply:
Agree. Added missing comments